joint_goal:
  joint_name: ["joint1", "joint2", "joint3", "joint4"]
  joint_positions: [-1, -1, -1, -1]
